src/pages/Selected/components/SelectSeat.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import * as S from './SelectSeat.styles';
+import { MdOutlineChairAlt } from 'react-icons/md';
+
+export default function SelectSeat() {
+  const [seats, setSeats] = useState([]);
+
+  useEffect(() => {
+    fetch('./data/seats.json')
+      .then(res => res.json())
+      .then(res => {
+        setSeats(res);
+      });
```

Comment:
```suggestion
      .then(data => {
        setSeats(data);
      });
```

통상적으로 res는 response의 약자로 통신 요청에 대한 응답에 대해 표시하는 약자입니다.
통신 후 첫번째 then 에서는 요청에 대한 응답이 왔기 때문에 그 응답을 js형태로 변경해주는 로직이고
그 데이터를 처리하는게 두번째 then이기에 위와 같이 해주는 것이 해당 매개변수의 역할을 제대로 나타내는 것이라고 보이네요!